src/Corax/IndexEntryBuilder.cs

```diff
@@ -141,7 +129,7 @@ public void Write(int field, ReadOnlySpan<byte> value, long longValue, double do
             int dataLocation = _dataIndex;
 
             // Write known field pointer.
-            _knownFieldsLocations[field] = dataLocation | IsTypedMask[sizeof(int)];
+            _knownFieldsLocations[field] = dataLocation | unchecked((int)0x80000000);
```

Comment:
Can you define constants here, instead of hard coded values?

Reply:
Yes, I was fighting the JIT there... this is a candidate for a constant.